test/app/App.js

```diff
@@ -114,6 +114,24 @@ describe('App', function() {
 		globals.window = window;
 	});
 
+	it('should add just path without queryString on findRoute when ignoringQueryString is enabled', () => {
```

Comment:
Adds test to check if the screen is being created for two navigations to same path with different query strings, e.g. `/path?foo=1` then `/path?foo=2`.